{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDomain"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/chatproviderssdk\/chatattachmenterror\/errordomain-49mx"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.zendesk.ChatProvidersSDK\/documentation\/ChatProvidersSDK\/ChatAttachmentError\/errorDomain-49mx","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The domain of the error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDomain"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"title":"errorDomain","roleHeading":"Type Property","role":"symbol","symbolKind":"property","externalID":"s:16ChatProvidersSDK0A15AttachmentErrorO11errorDomainSSvpZ","modules":[{"name":"ChatProvidersSDK"}]},"hierarchy":{"paths":[["doc:\/\/com.zendesk.ChatProvidersSDK\/documentation\/ChatProvidersSDK","doc:\/\/com.zendesk.ChatProvidersSDK\/documentation\/ChatProvidersSDK\/ChatAttachmentError"]]},"references":{"doc://com.zendesk.ChatProvidersSDK/documentation/ChatProvidersSDK":{"role":"collection","title":"ChatProvidersSDK","abstract":[],"identifier":"doc:\/\/com.zendesk.ChatProvidersSDK\/documentation\/ChatProvidersSDK","kind":"symbol","type":"topic","url":"\/documentation\/chatproviderssdk"},"doc://com.zendesk.ChatProvidersSDK/documentation/ChatProvidersSDK/ChatAttachmentError/errorDomain-49mx":{"role":"symbol","title":"errorDomain","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDomain"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"The domain of the error."}],"identifier":"doc:\/\/com.zendesk.ChatProvidersSDK\/documentation\/ChatProvidersSDK\/ChatAttachmentError\/errorDomain-49mx","kind":"symbol","type":"topic","url":"\/documentation\/chatproviderssdk\/chatattachmenterror\/errordomain-49mx"},"doc://com.zendesk.ChatProvidersSDK/documentation/ChatProvidersSDK/ChatAttachmentError":{"role":"symbol","title":"ChatAttachmentError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ChatAttachmentError"}],"abstract":[{"type":"text","text":"The error of type "},{"type":"codeVoice","code":"ChatAttachmentError"},{"type":"text","text":" is emitted when file upload has failed."}],"identifier":"doc:\/\/com.zendesk.ChatProvidersSDK\/documentation\/ChatProvidersSDK\/ChatAttachmentError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ChatAttachmentError"}],"url":"\/documentation\/chatproviderssdk\/chatattachmenterror"}}}